dev-setup/setup-ozone-s3.sh

```diff
@@ -5,5 +5,21 @@
 
 # Create namespace to deploy Ozone
 kubectl create namespace local-ozone | echo "local-ozone namespace already exists. Ok..."
-# Deploy Ozone resources
-kubectl -n local-ozone apply -f ozone
+
+if [ "$(uname -s)" = "Darwin" ] && [ "$(arch)" = "arm64" ]; then
+  # Replace Ozone docker image for MacOSX running on ARM64 architecture
+  X86_IMAGE="apache/ozone:1.2.1"
```

Comment:
this is good idea. instead of manipulating files like this, can you make the ozone resources a helm chart, and use helm install with values like `$ helm install --set image=leletan/ozone:1.2.1-20220623-1-aarch64`?

that way the scripts here will be much cleaner.

Reply:
Let me do some research and come back to this.